### ISSUE TITLE
Extension: opening from viewlet removes focus from viewlet

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#58487]
Steps to Reproduce:

1. click on an extension
2. => 🐛 focus lost so you are unable to keyboard navigate in the list

/cc @mjbvz
